sdks/python/setup.py

```diff
@@ -93,6 +93,10 @@ def get_version():
     'protorpc>=0.9.1,<0.12',
     'python-gflags>=2.0,<4.0.0',
     'pyyaml>=3.10,<4.0.0',
+    'pyhamcrest>=1.9,<2.0',
+    ]
+REQUIRED_TEST_PACKAGES = [
+    'pyhamcrest>=1.9,<2.0',
```

Comment:
I suggest that we keep this pyhamcrest and remove the one in REQUIRED_PACKAGES above.